build.sbt

```diff
@@ -75,3 +80,52 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     },
     consoleQuick / initialCommands := ""
   )
+
+lazy val scalacheck = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("scalacheck"))
+  .settings(
+    name := s"${projectName}-scalacheck",
+    libraryDependencies ++= Seq(
+      "org.scalacheck" %%% "scalacheck" % scalacheckV
+    ),
+    console / initialCommands := {
+      List(
+        "cats.",
+        "cats.syntax.all.",
+        "org.typelevel.idna4s.core.",
+        "org.typelevel.idna4s.core.syntax.all.",
+        "org.typelevel.idna4s.scalacheck.")
+        .map(value => s"import ${value}${wildcardImport.value}")
+        .mkString("\n")
+    },
+    consoleQuick / initialCommands := ""
+  )
+  .dependsOn(core)
+
+lazy val tests = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("tests"))
+  .settings(
+    name := s"${projectName}-tests",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "discipline-munit" % disciplineMunitV,
+      "org.typelevel" %%% "cats-laws"        % catsV
+    ),
+    console / initialCommands := {
+      List(
+        "cats.",
+        "cats.syntax.all.",
+        "org.typelevel.idna4s.core.",
+        "org.typelevel.idna4s.core.syntax.all.",
+        "org.typelevel.idna4s.scalacheck.")
+        .map(value => s"import ${value}${wildcardImport.value}")
+        .mkString("\n")
+    },
+    consoleQuick / initialCommands := ""
+  )
+  .jsSettings(
+    Test / scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule))
+  )
```

Comment:
This shouldn't be necessary.

Reply:
Ah. I needed that when I was using discipline-munit 1.0.9 to avoid this.

```
 [error] munit.internal.JSFs$ needs to be imported from module 'fs' but module support is disabled
 [error]   called from private java.nio.file.Files$.$anonfun$readAllBytes$1(java.nio.file.Path)[byte
 ```
 
 but I guess 2.0.0-M3 changes things so that's not needed. I'll remove it.

Reply:
Yes, actually that was fixed in:
- https://github.com/scalameta/munit/pull/376